src/Hazelcast.Net.Tests/FlakeId/BatchTests.cs

```diff
@@ -0,0 +1,93 @@
+﻿// Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Hazelcast.DistributedObjects.Impl;
+using Hazelcast.Testing;
+using NUnit.Framework;
+
+namespace Hazelcast.Tests.FlakeId
+{
+    using BatchTests_;
```

Comment:
Is this better than having a separate file?

Reply:
Ah, I am not too sure. Ideally I'd want a test to "pollute" the namespace as little as possible, so my first thought was, if a method or a class is used by one test only, let's keep it private to the test. But then, when it's an extension method, it cannot be nested inside the test class itself, so how do we achieve this? I have the same situation in other tests, where clearly the extension method only makes sense for one test, but it's convenient. This namespace hack is useful. But it's... not pretty. Maybe you can come up with a better idea?

We can:
* Move everything back to their own file.
* Use a better namespace name such as `BatchTestsDependencies` or something, and use the same pattern everywhere.
*  ?

I am not too sure and welcome suggestions / discussions.

Reply:
No ideas at the moment, maybe will figure something out later